mix.exs

```diff
@@ -50,7 +50,8 @@ defmodule Triplex.Mixfile do
     [
       {:ecto, "~> 2.1"},
       {:postgrex, ">= 0.11.0"},
-
+      {:mariaex, "~> 0.8.2"},
```

Comment:
So, let's make this one and `:postgrex` dependencies optional, since the user will probably choose between one and another.

Reply:
I kept only mariaex optional, because otherwise your test suite fails. It's weird, I would have expected the "optional" requirement to have an effect only if the project is included by another one: https://hexdocs.pm/mix/Mix.Tasks.Deps.html but I must be missing something.